code/extension/Member.php

```diff
@@ -26,6 +26,8 @@ class Member extends \DataExtension
 
     private static $admins_can_disable = false;
 
+    private static $regenerate_on_activation = true;
```

Comment:
Just to check my understanding, with this set to `true` here, this is the same as the current default behaviour (it's a non-breaking change). Setting to false is what is triggering the regenerate.